config/initializers/rolify.rb

```diff
@@ -0,0 +1,10 @@
+Rolify.configure do |config|
+  # By default ORM adapter is ActiveRecord. uncomment to use mongoid
+  # config.use_mongoid
+
+  # Dynamic shortcuts for User class (user.is_admin? like methods). Default is: false
+  # config.use_dynamic_shortcuts
+
```

Comment:
Explicitly NOT using dynamic shortcuts/built-in provided by rolify since they allow a User to have more than one role.  A user should ONLY have one role.

Reply:
I would say early stages yes, however I can think of times where as applications grow, individual users can have many roles (i.e. think of moderators in chats, they're still chatters themselves but also have the privileges of what moderating gives them). So certainly something to keep in mind as this application grows

Reply:
Good point, and I agree.  A user can potentially have a role as a :user for everywhere except in companies, where they may be a :moderator.  As it stands now we just have :user (default state) and :admin.  Absolutely can grow to have more roles.